build_bundle_image.sh

```diff
@@ -273,6 +275,13 @@ function push_docker_image {
 	fi
 }
 
+function append_lts_if_q1_release {
+	if [[ "${LIFERAY_DOCKER_RELEASE_VERSION}" == *.q1.* ]]
+	then
+		LIFERAY_DOCKER_RELEASE_VERSION="${LIFERAY_DOCKER_RELEASE_VERSION}-LTS"
+	fi
```

Comment:
Hey @davisaints, can you please make this logic to be applied starting on 2025.q1 and above only at least for now?

Thanks